JSDemos/Demos/Toolbar/Multiline/jQuery/styles.css

```diff
@@ -0,0 +1,54 @@
+.dx-resizable-handle::after {
+  content: "";
+  background-color: #337ab7;
+  position: absolute;
+  width: 12px;
+  height: 12px;
+  border: 1px solid #337ab7;
+  top: calc(50% - 7px);
+}
+
+.dx-resizable-handle-right::after {
+  right: -7px;
+}
+
+.dx-toolbar.dx-toolbar-multiline .dx-toolbar-item {
+  margin-bottom: 5px;
+}
+
+.dx-toolbar-menu-section .menu-separator-container {
```

Comment:
Let's rename 
- `.menu-separator-container` to `.toolbar-menu-separator-container`
- `.separator` to `.toolbar-separator`
- `.menu-separator` to `.toolbar-menu-separator`

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/9b1f7460ae9d90f96eef54b91e00e769c2a65696